apps/studio/components/interfaces/Database/Functions/DeleteFunction.tsx

```diff
@@ -8,7 +8,7 @@ import TextConfirmModal from 'ui-patterns/Dialogs/TextConfirmModal'
 interface DeleteFunctionProps {
   func?: DatabaseFunction
   visible: boolean
-  setVisible: (value: boolean) => void
+  setVisible: (value: string) => void
```

Comment:
Why string? boolean with the logic made much more sense no? This can lead to confusion on the DX side